packages/validator/src/cli-validator/run-validator.js

```diff
@@ -34,28 +36,78 @@ const processInput = async function(program) {
     return Promise.reject(2);
   }
 
+  // Set default loglevel of the root logger to be 'info'.
+  // The user can change this via the command line.
```

Comment:
Yeah, actually I was thinking about this from the opposite direction.  Like you said, if a user sets the root logger to error, then they won't see any normal validator output. But this might be exactly what they want. I was thinking of a scenario where the user might just want to check the return code from the validator and not clutter up the screen with messages (perhaps he's running the validator from a script or something).
I suspect that it will be rare for a user to change the root logger's level.